app/models/raw_input_changes.rb

```diff
@@ -41,13 +41,22 @@ def matching_organization
   end
 
   def matching_website
-    @matching_website ||= Website.find_by(content: url)
+    @matching_website ||= find_website(url) || find_website(resolved_url)
+  end
+
+  def find_website(content)
+    Website.find_by(content: content)
   end
 
   def url
     @attrs.fetch(:website, {})[:content]
   end
 
+  def resolved_url
+    @resolved_url ||= WebsiteResolver.resolve url
+    @attrs[:website][:content] = @resolved_url if @resolved_url
```

Comment:
Ok, you've convinced me - I'll work on refactoring this a bit to save both the `301` and the `200`. But I think I'd like to ignore the status part for this PR. If we end up wanting this info, I say we do that in a separate ticket. You good with that?